fiddler/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+fiddler.accuracy,gauge,,percent,,"The accuracy of the model",0,fiddler,accuracy,
+fiddler.prediction_drift,gauge,,percent,,"The prediction drift of the model",0,fiddler,prediction_drift,
```

Comment:
Can we add any additional expected metrics as collected from the `metrics_list`? 

```
metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
```

Reply:
@fanny-jiang I want to get one metric to start working properly and tested before I add more metrics. So lets keep it this way until we get a basic working piece.